### ISSUE TITLE
Thunkable Live App doesnt load data from Google Spreadsheet in the Data List View

### ISSUE THREAD

**Issue by muhammedjailam** [thunkable/thunkable-issues#783]
After adding Google Spreadsheet as a Data Source, and mapping the columns to the fields in the Data List View Thumbnail - The preview on the Thunkable web view works. But when trying to Live Test via the Thunkable Live App (for Android and on iOS) the data list view show real data. But shows placeholders for the data in each row. 

Then when we try to pull to refresh - the loading keeps loading forever. When exporting as APK for testing - the app same is seen. 

### Platform
Android companion and iOS companion. Both Apps using the latest versions. 

### Issue/Request Origin
Self Generated

### Steps to Reproduce
1. Create a Google Spreadsheet Data source that has multiple columns and few data rows
2. Add a Data List View Component and connect to the data source. Map the Columns to the columns.
3. Open the project on the Thunkable Live App. 

### Project Link
https://x.thunkable.com/copy/3342833c5f85a82f912d99fa7e4d180e

### Expected Behavior
The App display the content from the Google Spreadsheet is responsive to the changes made. 

### Actual Behavior
What actually happens?
The data is not displayed. Then when we try to pull to refresh - the loading keeps loading forever. When exporting as APK for testing - the app same is seen. 

![viber_image_2020-12-21_10-18-13](https://user-images.githubusercontent.com/9347984/102742098-e8bf0700-4375-11eb-99fa-ddfae0fdbafe.jpg)

**Comment by josmas** [comment]
hi @muhammedjailam is this the original app with the data source or a copy you have made of your own app?

Note that when you make copies, all data is replaced by placeholder data. This is so the app keeps working, but if you share your copy link with others (as you did in this report) your credentials are not exposed.

If that is the case, please see a discussion about this behaviour in this other report #654

**Comment by muhammedjailam** [comment]
@josmas Thank you for your response. 

This is my original project. I have used the same method to do a similar project just 2 days back, Then it worked like a charm. But after trying last night I found that the data was not loading on the app now. 

I would recommend if you try on your side and see if its working

**Comment by josmas** [comment]
Hi there, I do not see the placeholder data; but the app does not finish loading all the data either...

@StevenPurn any idea what might be going on? There were some fixes for this last week, but I am not sure if this has been deployed yet.

**Comment by muhammedjailam** [comment]
@josmas yes. I think my explanation might have had been a bit confusing. Deeply appreciate you looking into the matter. 
Would appreciate if the issue is fixed earliest.

**Comment by StevenPurn** [comment]
No, this is the first I've heard of this. I'll take a look though.

**Comment by StevenPurn** [comment]
I think we have a solution for this. We're testing it out now. It should be included in our next release though I'm unsure when that will be at this point.

**Comment by muhammedjailam** [comment]
Would be great if you can release if the Earliest Possible. Unable to make Google Spreadsheet apps as of now!

**Comment by muhammedjailam** [comment]
Any updates on the issue would be highly appreciated.

**Comment by StevenPurn** [comment]
Sorry it ended up taking so long! Over the holidays it gets much more difficult for any changes to go out since more people are away from the office and it takes more time for our apps to be reviewed by Google and Apple. We should be getting the fix out some time in the next couple of days but we're trying to get it released ASAP.
